CMakeLists.txt

```diff
@@ -49,12 +49,12 @@ Format: Auto | Common | All | LIST(ARCH_AND_PTX ...)
 - NUM: Any number. Only those pairs are currently accepted by NVCC though:
        2.0 2.1 3.0 3.2 3.5 3.7 5.0 5.2 5.3 6.0 6.2 7.0 7.2 7.5")
 option(USE_NCCL "Use NVidia NCCL with CUDA" OFF)
+option(USE_CUTENSOR "Use cuTENSOR" OFF)
 option(USE_OPENCV "Build with OpenCV support" ON)
 option(USE_OPENMP "Build with Openmp support" ON)
 option(USE_FATBIN_COMPRESSION "Compress nvcc fatbin output" ON)
 cmake_dependent_option(USE_NVML "Build with nvml support if found" ON "USE_CUDA" OFF)
 cmake_dependent_option(USE_CUDNN "Build with cudnn support" ON "USE_CUDA" OFF) # one could set CUDNN_ROOT for search path
-cmake_dependent_option(USE_CUTENSOR "Build with cuTENSOR support" ON "USE_CUDA" OFF) # one could set CUTENSOR_ROOT for search path
```

Comment:
Why revert this commit? The prior approach was correct?

Reply:
There is no need to revert.